docs/validator/validator-faq.md

```diff
@@ -47,7 +54,8 @@ No. However, the protocol measures the uptime of each validator, and if the gene
 
 ### What are the responsibilities of a validator?
 
-High level, validators must run node and be mostly online. However, it's very important to keep private keys safe, otherwise adversaries might use them to sign malicious blocks, and trigger the protocol slashing.
+High level, validators must run node and be mostly online. Also, they have to be constantly connected on the official [Slack chat](https://near.chat) in the `#community-validator-announcement` channel, in case of emergencies and upcoming hard forks.
```

Comment:
Do we want to switch to discord instead?

Reply:
done